runner/execer/fake/intercept.go

```diff
@@ -0,0 +1,31 @@
+package fake
```

Comment:
Yup, you're right. Ideas I have to solve it:
a) just accept it's a little odd
b) make a new package runner/execer/intercept that holds this
c) rename runner/execer/fake to a new package runner/execer/execers that holds the SimExecer and the InterceptExecer
d) something else better I haven't thought of (but hopefully you have!)

WDYT?